compute/k8s-services/vars.tf

```diff
@@ -619,6 +619,23 @@ variable "atlantis_staging_aws_secret" {
   description = "AWS Secret for staging environment"
 }
 
+variable "atlantis_resources_requests_cpu" {
+  type        = string
+  default     = "100m"
+  description = "CPU resources request size"
+}
+
+variable "atlantis_resources_requests_memory" {
+  type        = string
+  default     = "1536Mi"
+  description = "Memory resources requests size"
+}
+
+variable "atlantis_resources_limits_memory" {
+  type        = string
+  default     = null
```

Comment:
```suggestion
  default     = "1536Mi"
```

Reply:
This is on purpose. There is a local variable in the sub module that checks if resources_limits_memory is different than null or not. If it is null, then it will use the same value as for resources_requests_memory